crates/shared/src/http_solver/model.rs

```diff
@@ -249,6 +258,7 @@ pub struct ExecutedAmmModel {
     /// able to determine and order of execution. That is, solver may have a solution
     /// which it wants to share with the driver even if it couldn't derive an execution plan.
     pub exec_plan: Option<ExecutionPlan>,
+    pub fee: Option<u32>,
```

Comment:
Q: I need to inform quasimodo devs to add this field?

Reply:
Probably, seeing AFAIU, they need to provide this to us for encoding UniV3 swaps.

Reply:
I just realised - this fee shouldn't be needed with the `ExecutedAmm` model right? This value is constant for a given pool - so the HTTP solver shouldn't need to send it back.

Reply:
Yes, but currently there is no mapping between `ExecutedAmmModel` instance and the corresponding pool, for which the executed amounts are calculated.

Reply:
There should be though... otherwise this `ExecutedAmmModel` wouldn't be enough to encode BalancerV2 swaps (since they require encoding a pool ID).

Reply:
You are right. Fee is copied over to `UniswapV3SettlementHandler`.